docs/api.html

```diff
@@ -317,55 +301,51 @@
 </div>
 <div class="section" id="module-mabwiser.mab">
 <span id="mab"></span><h2>mab<a class="headerlink" href="#module-mabwiser.mab" title="Permalink to this headline">¶</a></h2>
-<table class="docutils field-list" frame="void" rules="none">
-<col class="field-name" />
-<col class="field-body" />
-<tbody valign="top">
-<tr class="field-odd field"><th class="field-name">Author:</th><td class="field-body">FMR LLC</td>
-</tr>
-<tr class="field-even field"><th class="field-name">Email:</th><td class="field-body"><a class="reference external" href="mailto:mabwiser&#37;&#52;&#48;fmr&#46;com">mabwiser<span>&#64;</span>fmr<span>&#46;</span>com</a></td>
-</tr>
-<tr class="field-odd field"><th class="field-name">Version:</th><td class="field-body">1.9.1 of May 27, 2020</td>
-</tr>
-</tbody>
-</table>
+<dl class="field-list simple">
+<dt class="field-odd">Author</dt>
+<dd class="field-odd"><p>FMR LLC</p>
+</dd>
+<dt class="field-even">Email</dt>
+<dd class="field-even"><p><a class="reference external" href="mailto:mabwiser&#37;&#52;&#48;fmr&#46;com">mabwiser<span>&#64;</span>fmr<span>&#46;</span>com</a></p>
+</dd>
+<dt class="field-odd">Version</dt>
+<dd class="field-odd"><p>1.10.0 of June 22, 2020</p>
```

Comment:
Not a big deal, but for some reason this doc didn't update correctly?

Reply:
Sphinx only recompiles the api page if something in the docstrings changes

Reply:
Got it.

Reply:
(still wanna keep the mabwiser email?)

Reply:
@skadio I forgot to make the update for this version and PyPi doesn't allow you to update existing version files so we would have to increment to 1.10.2. I've got a few cleanup items to do another patch for soon, so the email will get updated in that, and for now we can keep this pr focused on the PyPi release.